website/docs/gitops-run/get-started.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Get Started
+sidebar_position: 2
+hide_title: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Get Started with GitOps Run
+
+Gitops Run supports two different modes of operation - directly on a
+cluster, or as sandboxed sessions. The sandboxed sessions are intended
```

Comment:
Do we need the comma before "or" here? It just looks like a simple list made of two items to me (but Ukrainian/Russian punctuation is completely different, so I may be wrong).